docs/configuration.md

```diff
@@ -300,6 +300,29 @@ node-local load balancing.
 | `apiServerBindPort`          | Port number on which to bind the Envoy load balancer for the Kubernetes API server to on a worker's loopback interface.  Default: `7443`. |
 | `konnectivityServerBindPort` | Port number on which to bind the Envoy load balancer for the konnectivity server to on a worker's loopback interface. Default: `7132`.    |
 
+##### `spec.network.controlPlaneLoadBalancing`
+
+Configuration options related to k0s's [control plane load balancing] feature
+
+| Element         | Description                                                                               |
+| --------------- | ----------------------------------------------------------------------------------------- |
+| `vrrpInstances` | Configuration options related to the VRRPInstances. This is used to configure virtual IPs |
```

Comment:
Based on this explanation and the `VRRPInstances` below, I'm not really sure how I'd use this. E.g. is `vrrpInstances` an array as the name hints?

Maybe some simple example would help users.

Reply:
I added some more text to make it clearer, I hope this is better.